sbin/pfdns

```diff
@@ -137,6 +138,12 @@ foreach my $network ( keys %ConfigNetworks ) {
     }
 }
 
+# Instantiate dns_hash
+my %dns_hash;
```

Comment:
I suggest you create a pfconfig resource for that and have it expired by config::Domain

This way, we won't need to restart pfdns when we activate the passthrough for this domain